DQM/L1TMonitor/src/L1TObjectsTiming.cc

```diff
@@ -325,71 +806,96 @@ void L1TObjectsTiming::analyze(const edm::Event& e, const edm::EventSetup& c) {
   // Open uGT readout record
   edm::Handle<GlobalAlgBlkBxCollection> uGtAlgs;
   e.getByToken(l1tStage2uGtProducer_, uGtAlgs);
-          
+
   // Filling eta-phi map for muons for BX=-2,-1,0,+1,+2
   for (int itBX = MuonBxCollection->getFirstBX(); itBX <= MuonBxCollection->getLastBX(); ++itBX) {
-    for (l1t::MuonBxCollection::const_iterator Muon = MuonBxCollection->begin(itBX); Muon != MuonBxCollection->end(itBX); ++Muon) {
+    for (l1t::MuonBxCollection::const_iterator Muon = MuonBxCollection->begin(itBX);
+         Muon != MuonBxCollection->end(itBX);
+         ++Muon) {
       if (Muon->pt() >= muonPtCut_ and Muon->hwQual() >= muonQualCut_) {
         denominator_muons->Fill(Muon->eta(), Muon->phi());
-        int index = (int)itBX - std::min(0, 1 - (int)bxrange_%2 - (int)std::floor(bxrange_/2.)); // the correlation from itBX to respective index of the vector
+        int index = (int)itBX -
+                    std::min(0,
+                             1 - (int)bxrange_ % 2 -
+                                 (int)std::floor(bxrange_ /
+                                                 2.));  // the correlation from itBX to respective index of the vector
```

Comment:
I would say the formatting script gets a bit over-excited here. This makes the readability worse in my opinion.

Reply:
@smuzaffar @davidlange6 is this the expected result?

Reply:
@thomreis , I agree. Looks like it is trying to keep the line length under 120 characters. O would suggest that we move the comment on the new line so that clang-format do not break this line.